boa/src/syntax/ast/node/object/mod.rs

```diff
@@ -53,24 +56,32 @@ impl Object {
         indent: usize,
     ) -> fmt::Result {
         f.write_str("{\n")?;
+        let indentation = "    ".repeat(indent + 1);
         for property in self.properties().iter() {
             match property {
                 PropertyDefinition::IdentifierReference(key) => {
-                    write!(f, "{}    {},", indent, key)?;
+                    writeln!(f, "{}{},", indentation, key)?;
                 }
                 PropertyDefinition::Property(key, value) => {
-                    write!(f, "{}    {}: {},", indent, key, value)?;
+                    dbg!(&value);
```

Comment:
Forgotten `dbg!()`?

Reply:
Whoops. Thats removed now. The Object definition has been more complicated than I thought, so I've added a couple extra `dbg!()` calls here and there.